views/Products/Create/Form.tsx

```diff
@@ -106,13 +119,14 @@ const CreateProductForm: FC<MutableProductFormProps> = ({ onSubmit, loading }) =
         </div>
       </div>
       {errors.imageUrl && <small className={stylesInput['error-message']}>{errors.imageUrl.message}</small>}
-
-      <button className="save" type="submit">
-        {loading ? 'Guardando' : 'Guardar'}
-      </button>
-      <button className="delete" onClick={() => reset()}>
-        Eliminar
-      </button>
+      <div className="button-container">
+        <Button className={stylesInput['button-save']} type="submit">
+          {loading ? 'Guardando' : 'Guardar'}
+        </Button>
```

Comment:
What about disable the submit button while the form is loading?

```suggestion
        <Button disabled={loading} className={stylesInput['button-cancel']} onClick={() => router.push('/productos')} disabled={loading}>
          Cancelar
        </Button>
```